kafka/hbase-kafka-proxy/src/main/java/org/apache/hadoop/hbase/kafka/KafkaBridgeConnection.java

```diff
@@ -187,6 +187,14 @@ public TableBuilder setWriteRpcTimeout(int timeout) {
         return null;
       }
 
+      /**
+       * This method was added in HBASE-27657. We donot add @Override to allow code to compile with
+       * versions of hbase both having and not having HBASE-27657
+       **/
+      public TableBuilder setRequestAttribute(String key, byte[] value) {
```

Comment:
This is a fragile approach. This repository needs sureing up if it is going to intentionally support multiple minor versions of HBase. At the minimum, we need an HBase versions check in our pre-commit and nightlies, similar to what HBase has for Hadoop.